parcels/fieldfilebuffer.py

```diff
@@ -105,6 +106,9 @@ def lonlat(self):
         else:
             xdim = lon.size if len(lon.shape) == 1 else lon.shape[-1]
             ydim = lat.size if len(lat.shape) == 1 else lat.shape[-2]
+            if self.gridindexingtype in ["croco"]:
+                xdim -= 1
+                ydim -= 1
```

Comment:
I guess this is due to how croco is represented under the hood?

Reply:
Yes exactly; the last row and column can't be used in croco